test/Microsoft.IdentityModel.Tokens.Tests/TokenValidationResultTests.cs

```diff
@@ -45,8 +45,8 @@ public void GetSets()
             TokenValidationResult tokenValidationResult = new TokenValidationResult();
             Type type = typeof(TokenValidationResult);
             PropertyInfo[] properties = type.GetProperties();
-            if (properties.Length != 7)
-                Assert.True(false, "Number of public fields has changed from 7 to: " + properties.Length + ", adjust tests");
+            if (properties.Length != 9)
```

Comment:
+2 properties but +1 test case. Should Claims not be included here?

Reply:
I believe every time a new object is created when accessing Claims, [here ](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/92580d7f1a3027c6f400cc11be46d9e061dedea2/test/Microsoft.IdentityModel.TestUtils/TestUtilities.cs#L161) they don't match on all properties as they are different objects.